VizNetworkApp/Requests Manager/VizNetworkManager.swift

```diff
@@ -55,10 +50,16 @@ class VizNetworkManager {
                     group.leave()
                 }
             }
-            
-            DispatchQueue.global().async { // todo: private queue with label
-                task.resume()
+            if let dispatchQueue = dispatchQueue {
```

Comment:
just use `dispatchQueue: DispatchQueue? = .main` in the signature and lose this

Reply:
default was .global() - changed to that